MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+include src/humanize/py.typed
+include src/humanize/*.pyi
```

Comment:
This project is using https://github.com/pypa/setuptools_scm, which means stuff in SCM/Git is already included.

`MANIFEST.in` is only needed to included non-SCM files or to exclude SCM files.

So let's remove this file.

Reply:
Ah yeah, you did mention that before, and I forgot. 😅